### ISSUE TITLE
TypeError: control.syncContainer is undefined

### ISSUE THREAD

**Issue by joyously** [xwp/wp-core-media-widgets#189]
http://localhost/review/wp/wp-content/plugins/wp-core-media-widgets/wp-admin/js/widgets/media-gallery-widget.js?ver=4.7.5 line 194

**Comment by westonruter** [comment]
@joyously Please provide more details. I cannot reproduce this on the widgets admin screen (even in accessibility mode) or in the Customizer.

**Comment by joyously** [comment]
Well, I thought the error message said it all. WordPress 4.7.5, Gallery widget on admin widgets page. 
I got this error when I had chosen 3 images and clicked on Insert Gallery.
The Gallery widget shows as blank saying "No images selected", but when I click on that (after getting the error), it takes me back to the media modal, where my 3 images are still selected and the button now says "Update Gallery". Clicking this gives the same error as before.

**Comment by westonruter** [comment]
@joyously oh, WordPress 4.7.5? That is likely the problem then. It's only been tested on 4.8.x – 4.9-alpha, as far as I know. Since we're not implementing here for compatibility with version prior to 4.9-alpha (since that is the version we'd be merging into), please test in 4.8 since that is when media widgets were introduced in core. I'll bump the minimum required version to 4.8.

**Comment by joyously** [comment]
Hmm, I figured it would be something like that. But the previous version of this Core Widget plugin worked on 4.7.5. I suppose you had to remove the parts that got merged into core.

**Comment by westonruter** [comment]
Going to close this since it's not a 4.8+ issue, apparently.
